lib/buildConnectorsJson/index.js

```diff
@@ -94,9 +123,12 @@ module.exports = function (directory, config) {
 
 	});
 
+	if (_.isString(directory)) {
+		fs.writeFileSync(directory+'/connectors.json', JSON.stringify(connectorsJson, null, '\t'));
+	}
```

Comment:
not a big deal, but maybe this could be moved outside, so the connector builder does not generate the file

Reply:
This won't get generated because I'm passing `null` as `directory` in the `generateJsonSchema` method.

https://github.com/trayio/falafel/blob/CPS-54/f/expose-schema-generation-as-method/lib/index.js#L45